prow/jobs/config.yaml

```diff
@@ -18707,6 +18707,7 @@ periodics:
       - "--auto-release"
       - "--release-gcs"
       - "knative-releases/eventing-kafka"
+      - "knative-releases/eventing-ceph"
```

Comment:
This shouldn't be here - it's part of the `ci-knative-sandbox-eventing-kafka-auto-release` job